specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -1800,6 +1804,10 @@
             "modelAsString": true
           }
         },
+        "purchasePlan": {
+          "$ref": "#/definitions/PurchasePlan",
+          "description": "Purchase Plan Information of the Virtual Machine persisted on the Disk."
```

Comment:
if this is snapshot say that purchase plan information copied from source disk/snapshot

Reply:
Purchase plan information for the image from which the source disk for the snapshot was originally created.